### PR TITLE
Test change

### PR DESCRIPTION
**Pull request by akeithball**

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -26,3 +26,6 @@ log4j.rootLogger=WARN, stdout
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d [%-15.15t] %-5p %-30.30c{1} - %m%n
+
+# Another test comment
+#log4j.logger.org.apache.camel=DEBUG
```
Comment (cloudbotinmarsat) [inline src/main/resources/log4j.properties]: MAJOR Remove this commented out code. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=jproperties%3Acommented-out-code)

**Comment by cloudbotinmarsat** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (hammerdj1) [inline src/main/resources/log4j.properties]: please remove

File: src/main/resources/log4j.properties [hunk 2]
```diff
@@ -26,3 +26,9 @@ log4j.rootLogger=WARN, stdout
...
+
+# Another test comment
+#log4j.logger.org.apache.camel=DEBUG
```
Comment (cloudbotinmarsat) [inline src/main/resources/log4j.properties]: MAJOR Remove this commented out code. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=jproperties%3Acommented-out-code)

**Comment by cloudbotinmarsat** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Pull request closed without merging**